Assets/Events/Adventurers Leave.json

```diff
@@ -151,16 +150,16 @@
     {
         "name": "touch-of-gold",
         "headline": "Touch of Gold",
-        "article": "A disgruntled adventurer looking to make quick buck has blessed your people with gold... Made of those same people! One of our heroic adventurers jumped on the madman, turning themselves into gold, and crushing them in the process. Try as we might, we couldn't unfreeze the lost. We mourn as we melt and sell off all this new gold we found.",
+        "article": "A disgruntled traveller looking to make quick buck has blessed the townsfolk with gold... by turning them into it! One of our heroic adventurers jumped atop the madman, accidentally turning themselves into gold and crushing the culprit in the process. Try as we might, we couldn't unfreeze the lost, so we mourn as we melt down those brave souls that lost their lives - it's what they would have wanted us to do!",
         "image": "two-coins",
         "type": "AdventurersLeave",
         "outcomes":
         [
             {
                 "type": "AdventurersRemoved",
-                "customDescription": "Both adventurers have been lost",
+                "customDescription": "An adventurer has been lost",
                 "anyGuild": true,
-                "count": 2,
+                "count": 1,
```

Comment:
Is there a reason you made this 1 instead of 2? This is an article when adventurers are unhappy, so i think having it be an adventurer getting goldified makes more sense

Reply:
Big problem with it was that it was unclear throughout the text which adventurer was being referred to. I found it very hard to rewrite in such a way that was clear in that regard.